examples/examples.ipynb

```diff
@@ -47,16 +47,7 @@
    "cell_type": "code",
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #4.    pyro_sim.run_sim;
</pre><p>This should be <code>pyro_sim.run_sim();</code></p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/python-hydro/pyro2/pull/231/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='14'/>

Reply:
https://app.reviewnb.com/python-hydro/pyro2/pull/231/discussion/#comment-1740205658